scripts/fillTestTribesData.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+var path = require('path'),
+    mongooseService = require(path.resolve('./config/lib/mongoose')),
+    chalk = require('chalk'),
+    faker = require('faker'),
+    mongoose = require('mongoose');
+
+require(path.resolve('./modules/tribes/server/models/tribe.server.model'));
```

Comment:
We can probably remove these if we're already doing `mongooseService.loadModels` at line 60?

`loadModels` does import these files already for us:
https://github.com/Trustroots/trustroots/blob/698f2357ae5d07d5a422a2ce5780d8f13fda00f9/config/lib/mongoose.js#L13-L23

That change might not play nice with `mongoose.model('Tribe')` line tho so feel free to ignore; we can improve structure of these scripts in later iteration.

Reply:
Thanks! This is good to know about, but I'm going to hold off for a later iteration. I would like to make the command line script dumb regarding the DB and just call library methods that add the mock data. Right now the code is a bit interspersed.

Reply:
Also, I went ahead and rebased and added documentation. Let me know if you see anything else! Thanks!